samples/v1/test/cloudscheduler.test.yaml

```diff
@@ -0,0 +1,18 @@
+type: test/samples
+schema_version: 1
+test:
+  suites:
+  - name: Manage jobs
+    cases:
+
+    - name: cloud_scheduler_list_job
+      spec:
+      - call:
+          sample: cloud_scheduler_list_jobs
+          params:
+            project_id:
+              literal: "tribal-cortex-199921"
```

Comment:
Curious why this needs to be hard coded in the yaml.  Can we use `process.env.GCLOUD_PROJECT` that's already configured as part of the CI suite?

Reply:
Doesn't need to be hard coded, just easier while we're prototyping.